import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/CStoreService.java

```diff
@@ -170,7 +170,8 @@ protected void store(
         MonitoringService.addEvent(Event.CSTORE_BYTES, countingStream.getCount());
       } catch (DicomWebException e) {
       if (backupUploadService != null) {
-        reportBackupError(e);
+        MonitoringService.addEvent(Event.CSTORE_BACKUP_ERROR);
```

Comment:
We don`t need send event here. By contract - we sent events only when something finally failed. Not per every  resent attempt.